client/src/App.js

```diff
@@ -15,9 +16,14 @@ function App() {
     <div className='card-container'>
     <FlashCard  />
     <div className='card-controls'>
-    <Button text="Back"/>
-    <Button text="Flip"/>
-    <Button text="Next"/>
+      {controls.map((control) => {
+        return <Button>{control}</Button>
+      })}
+      <Button text="Back"/>
```

Comment:
Lines 22-24 and 19-21 are redundant. Was one of them working and another not?

Reply:
Lines 19-21 were not working properly--the buttons didn't show up correctly.